src/ecommerce.interfaces.ts

```diff
@@ -84,6 +84,16 @@ export interface SDKECommerceAPI extends IECommerceShared {
         transactionAttributes?: TransactionAttributes,
         eventOptions?: SDKEventOptions
     ): void;
+    logProductActionV2(
+        productActionType: valueof<typeof ProductActionType>,
+        product: SDKProduct | SDKProduct[],
+        options?: {
+            attrs?: SDKEventAttrs;
```

Comment:
I think that calling this `attrs` or `attributes` may be confusing since we have `attributes` as part of `selectPlacements`.

```suggestion
            eventAttributes?: SDKEventAttrs;
```